db/schema.rb

```diff
@@ -396,6 +396,7 @@
     t.bigint "user_id"
     t.datetime "verified_at"
     t.index ["user_id", "type_of"], name: "index_email_authorizations_on_user_id_and_type_of", unique: true
+    t.index ["user_id"], name: "index_email_authorizations_on_user_id"
```

Comment:
I have no idea why it's here...

Reply:
this should be removed from the PR, I'll investigate further later

Reply:
As @rhymes mentioned, just remove this as it's not related to the PR.